.jsonschema.json

```diff
@@ -680,7 +680,7 @@
         "pattern": "^https://github\\.com/(?!(features/actions)|(sponsors)|(logos)$)"
       },
       "then": {
-        "pattern": "^https://github\\.com/[^/]+/[^/]+/(blob/[a-f\\d]{40}/[^\\s]+)|(tree/[a-f\\d]{40}(/[^\\s]+)?)|(((issues)|(pull))/\\d+#issuecomment-\\d+)$"
+        "pattern": "^https://github\\.com/[^/]+/[^/]+/(blob/[a-f\\d]{40}/[^\\s]+)|(tree/[a-f\\d]{40}(/[^\\s]+)?)|(((issues)|(pull)|(discussions))/\\d+#(issuecomment|discussioncomment)-\\d+)$"
```

Comment:
@simple-icons/maintainers - can someone confirm the change to the JSON schema is okay for merging? Rest of the PR is fine.